README.md

```diff
@@ -220,9 +220,9 @@ Considering the following snippet as a visual example to the previous statement:
 The end result of such initialization will lead to bugs. 
 The `SDKName()` function, undefined at this point, executes before it becomes available in the environment's global variable. The script is not loaded yet.
 
-To make it work, some tricks are necessary to make sure the script execute with success. The event will store in the `SDKName.q` queue array. The SDK should be able to handle and execute `SDKName.q` event and initialize the `SDKName` namespace.
+To make it work, some tricks are necessary to make sure the script executes successfully. The event will (need to) be stored in the `SDKName.q` queue array. The SDK should be able to handle and execute the `SDKName.q` event and initialize the `SDKName` namespace.
 
-The following snippet depicts the statement in previous paragraph.
+The following snippet shos this:
```

Comment:
"shos" is an error, It should be "shows".

It would also be OK to leave initial phrase stand, unless there is a confusion it incur to the current context
"The following snippet depicts the statement in previous paragraph."

Reply:
agree. let's keep the original phrase.

Reply:
I think 'Keep it simple' ... for the majority to understand. Some may not know what depict means.
but as you wish ..